class/UserAccessManager.php

```diff
@@ -155,6 +155,9 @@ public function getFromCache($sKey)
     public function getUser($sId)
     {
         if (!isset($this->_aUsers[$sId])) {
+            if (!function_exists('get_userdata')) {
```

Comment:
Now I am really confused. It seems, that wp-includes/pluggable.php is always loaded.
http://wordpress.stackexchange.com/questions/39232/why-plugins-are-loaded-prior-to-pluggables
Is there a chance, that getUser($iId) is called at plugin initialization?
If this is the case, there is no need to include wp-includes/pluggable.php
it could actually cause problems, if I understand correctly. Each plugin is allowed to overwrite (implement) functions that are in pluggable. But it user-access-manager includes it, and a plugin that loads it later wants to overwrite  function, the behaviour would be undefined for all other plugins depending on this very plugins way to handle a predefined (or not predefined) function.